### ISSUE TITLE
The IconicsImageView  setColor method  can't change icon color

### ISSUE THREAD

**Issue by onloadcc** [mikepenz/Android-Iconics#307]
This method can't change icon color:
```xml
 IconicsImageView viewById = (IconicsImageView) findViewById(R.id.icon_image_view_android);
   viewById.setColor(ContextCompat.getColor(this, R.color.colorPrimary));
```
must use the method
```xml
IconicsImageView viewById = (IconicsImageView) findViewById(R.id.icon_image_view_android);
    viewById.getIcon().color(ContextCompat.getColor(this,R.color.colorPrimary));
```

This is bug or other reason?

**Comment by mikepenz** [comment]
@onloadcc it looks like a bug. We are looking into it

**Comment by mikepenz** [comment]
@onloadcc for now you can use the second variant. I'll let you know as soon as it's fixed

**Comment by mikepenz** [comment]
@onloadcc this is fixed in v2.9.1
